test/ArborX_EnableDeviceTypes.hpp.in

```diff
@@ -14,8 +14,8 @@
 
 #include <Kokkos_Core.hpp>
 
-#include <tuple>
+#include <boost/mpl/list.hpp>
 
-using ARBORX_DEVICE_TYPES = std::tuple<@ARBORX_DEVICE_TYPES@>;
+using ARBORX_DEVICE_TYPES = boost::mpl::list<@ARBORX_DEVICE_TYPES@>;
 
```

Comment:
Please include `BoostTest_CUDA_clang_workarounds.hpp` instead.

Reply:
Here you go!